cordformation/src/main/kotlin/net/corda/plugins/Cordformation.kt

```diff
@@ -66,6 +66,7 @@ class Cordformation : Plugin<Project> {
             createCompileConfiguration("cordapp", this)
             val cordaRuntime = createRuntimeConfiguration("cordaRuntime", this)
             createChildConfiguration(CORDFORMATION_TYPE, cordaRuntime, this)
+            createRuntimeConfiguration("cordaDriver", this)
```

Comment:
I don't think you want to be using `createRuntimeConfiguration` here because it's equivalent to:
```gradle
configurations {
    runtime.extendsFrom cordaDriver
}
```
i.e. anything added to the `cordaDriver` configuration will automatically be added to the `runtime` configuration too :scream_cat: !

Reply:
Code changed to address this comment

Reply:
Eh? I wasn't aware that I'd submitted that review yet... :confused: